### PR TITLE
add docs for the cluster

### PR DESCRIPTION
**Pull request by TomorrowIsAnOtherDay**
### This PR:
- add docs for the cluster

You can view the docs in [here](http://172.18.202.26:8004).
image

### CODE REVIEW

File: docs/parallel_training/recommened_pratice.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
+Recommened Practice
```
Comment (zenghsh3) [inline docs/parallel_training/recommened_pratice.rst]: spelling error: Recommended （filename is also wrong）

File: docs/parallel_training/setup.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+We use the port `6006` for demonstration, and it can be any available port.
+
+**Explaination**
+
+1. start a master node to manage computation resources.
+2. start a worker node that provides computation resources for the master. A worker will use all the CPUs by default. If you wish to specify the number of CPUs to be used, run the command with ``--cpu_num <cpu_num>`` (e.g.------cpu_num 10). 
+
+Note that the command ``xparl connect`` can be run at any time, at any machine to add more CPUs to the cluster.
+
+Here we give an example demonstating how to use ``@parl.remote_class`` for parallel computation.
```
Comment (zenghsh3) [inline docs/parallel_training/setup.rst]: spelling error: demonstrating

File: docs/parallel_training/setup.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+  xparl start --port 6006
+
+  xparl connect --address localhost:6006
+
+We use the port `6006` for demonstration, and it can be any available port.
+
+**Explaination**
+
+1. start a master node to manage computation resources.
```
Comment (zenghsh3) [inline docs/parallel_training/setup.rst]: Should we tell user that when we start a master, we will also start a worker in the same machine by default.

**Review by zenghsh3 — CHANGES_REQUESTED** [review]

File: docs/parallel_training/setup.rst (hunk 3 above)
Comment (TomorrowIsAnOtherDay) [reply docs/parallel_training/setup.rst]: Currently we are not adding any worker when we execute ``xparl start``..

File: docs/parallel_training/recommened_pratice.rst (hunk 1 above)
Comment (TomorrowIsAnOtherDay) [reply docs/parallel_training/recommened_pratice.rst]: done

File: docs/parallel_training/setup.rst (hunk 2 above)
Comment (TomorrowIsAnOtherDay) [reply docs/parallel_training/setup.rst]: done

File: docs/parallel_training/setup.rst (hunk 3 above)
Comment (TomorrowIsAnOtherDay) [reply docs/parallel_training/setup.rst]: Done.

File: docs/parallel_training/recommened_pratice.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        th.start()
+        threads.append(th)
+    for th in threads:
+        th.join()
+
+.. image:: ./elapsed_time.jpg
+  :width: 400px
+  :align: center
+
+| Only **3.74** seconds is needed !!! As you can see from the code above, it is the ``@parl.remote_class`` that makes the change happen. By simply adding this decorator, we achieved real multi-thread computation.
```
Comment (banma12956) [inline docs/parallel_training/recommened_pratice.rst]: typo: is needed -> are needed

File: docs/parallel_training/recommened_pratice.rst [hunk 5]
```diff
@@ -0,0 +1,75 @@
+Recommened Practice
+---------------------
+
+.. image:: ./poster.png
+  :width: 200px
+  :align: center
+| This tutorial shows how to use ``@parl.remote_class`` to implement parallel computation with **multi-threads**.
+
+| PYTHON has poor performance in multi-threading because of the `GIL <https://realpython.com/python-gil/>`_ , and we always find that multi-thread programming in PYTHON cannot bring any benefit of the running speed, unlike other program languages such as ``C++`` and ``JAVA``.
```
Comment (banma12956) [inline docs/parallel_training/recommened_pratice.rst]: typo: benefit -> benefits

**Review by banma12956 — DISMISSED** [review]

File: docs/parallel_training/recommened_pratice.rst (hunk 5 above)
Comment (TomorrowIsAnOtherDay) [reply docs/parallel_training/recommened_pratice.rst]: done

File: docs/parallel_training/recommened_pratice.rst (hunk 4 above)
Comment (TomorrowIsAnOtherDay) [reply docs/parallel_training/recommened_pratice.rst]: done

**Review by zenghsh3 — DISMISSED** [review]
LGTM

**Review by zenghsh3 — DISMISSED** [review]

File: docs/parallel_training/recommended_pratice.rst [hunk 6]
```diff
@@ -0,0 +1,75 @@
+Recommended Practice
```
Comment (zenghsh3) [inline docs/parallel_training/recommended_pratice.rst]: spelling error in filename: pratice -> practice

Comment (TomorrowIsAnOtherDay) [reply docs/parallel_training/recommended_pratice.rst]: done

**Review by zenghsh3 — APPROVED** [review]

**Pull request merged**